### PR TITLE
refactor: add blank lines into the controller file

### PR DESCRIPTION
**Pull request by pinakam**

### CODE REVIEW

File: app/Http/Controllers/API/AuthController.php [hunk 1]
```diff
@@ -24,8 +24,10 @@ public function login(Request $request)
         if (Auth::attempt($data)) {
             $user = Auth::user();
 
+
```
Comment (SanjayMoweb) [inline app/Http/Controllers/API/AuthController.php:27]: Please remove this unwanted line first

File: app/Http/Controllers/API/AuthController.php [hunk 2]
```diff
@@ -24,8 +24,10 @@ public function login(Request $request)
...
             $token = $user->createToken('Laravel-9-Passport-Auth')->accessToken;
 
+
```
Comment (SanjayMoweb) [inline app/Http/Controllers/API/AuthController.php:30]: Also remove this blank space

**Review by SanjayMoweb — CHANGES_REQUESTED** [review]
Not approved, please make changes

File: app/Http/Controllers/API/AuthController.php (hunk 1 above)
Comment (pinakam) [reply app/Http/Controllers/API/AuthController.php]: I've remove this blank line

**Review by SanjayMoweb — APPROVED** [review]

**Pull request merged**